bindings/rust/evmc-vm/src/lib.rs

```diff
@@ -759,6 +759,7 @@ mod tests {
             block_timestamp: 235117,
             block_gas_limit: 105023,
             block_difficulty: Uint256 { bytes: [0xaa; 32] },
+            chain_id: Uint256 { bytes: [0u8; 32] },
```

Comment:
Can use `Uint256::default()` here, the other used the in-line declaration to set a non-zero value.

Reply:
I've changed it, though `tx_gas_price`, `tx_origin`, `block_coinbase` are zeroes here, too.